CHANGELOG.md

```diff
@@ -15,8 +15,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fall back to default for `WASMER_DIR` env var if it doesn't exist
 - Global install packages from the registry with `-g`/`--global` flag.
   - Packages are installed into `WASMER_DIR/globals`.
+  - Packages are runnable from any directory that does not already have that package installed.
```

Comment:
I like this update to the changelog.  I'm sure this will require restructuring how things work, but we could always print a message saying this and/or have a flag for overriding.   (I don't think these are high priority, we can add them to the backlog if they seem useful though)